### DISCUSSION TITLE
AG not connecting - Cubase 11 Win 10

### DISCUSSION THREAD

**Discussion by MrPisquan** [apohl79/audiogridder#566] [General]
Hi Andreas,
it's been few days that I cannot find a way to connect AudioGridder to Cubase.
This kind of problem happened the first time I updated AG to the new version. Then the problem disappeared after restarting the PC. Now unfortunately it's not the case.
From the zipped images you will see the weirdness of the situation.
Thank you for your support and for your amazing job! 
Federico

[AudioGridderPlugin_2021-05-18_20-54-30.log](https://github.com/apohl79/audiogridder/files/6504020/AudioGridderPlugin_2021-05-18_20-54-30.log)
[AudioGridderPlugin_2021-05-18_20-55-27.log](https://github.com/apohl79/audiogridder/files/6504021/AudioGridderPlugin_2021-05-18_20-55-27.log)
[AudioGridderServer_2021-05-18_20-47-44.log](https://github.com/apohl79/audiogridder/files/6504041/AudioGridderServer_2021-05-18_20-47-44.log)

[itsy.zip](https://github.com/apohl79/audiogridder/files/6504164/itsy.zip)

**Comment by MrPisquan** [comment]
Hi,
I think it has something to do with VST connect. It was active all the time. Once I was done with it and shut down VST connect, I restarted the system and AG is connecting now.
Is it possible that they share the same resources? 
Thank you, you are doing an amazing job!
Federico

**Reply by apohl79** [reply]
I have never looked at/used vst connect so I can't say. But as the connection problems went away w/o it, there might be something conflicting.
